src/test/java/com/softserve/edu/greencity/ui/tests/GreenCityTestRunner.java

```diff
@@ -29,14 +26,17 @@ public void beforeSuite() {
     }
 
     @BeforeClass
-    public void setUpBeforeClass() throws Exception {
-        driver = new ChromeDriver();
+    public void setUpBeforeClass() {
+        driver = new ChromeDriver(new ChromeOptions().addArguments("headless"));
```

Comment:
rewrite it in such case
                ChromeOptions chromeOptions = new ChromeOptions();
                chromeOptions.setHeadless(true);

Reply:
Thank you. 
Fixed.